source/views/settings/screens/overview/odds-and-ends.js

```diff
@@ -32,7 +36,13 @@ export class OddsAndEndsSection extends React.Component<Props> {
 				<ConnectedNotificationsCell onPress={this.onNotificationsButton} />
 
 				{isDevMode() && (
-					<PushButtonCell onPress={this.onDebugButton} title="Debug" />
+					<>
+						<PushButtonCell onPress={this.onDebugButton} title="Debug" />
+						<PushButtonCell
```

Comment:
Once the other "Debug Section" for settings is merged in, this will move into that file.